ludwig/models/llm.py

```diff
@@ -49,7 +48,29 @@ def __init__(
         self.curr_device = torch.device("cpu")  # model initially loaded onto cpu
         logger.info("Done.")
 
-        self.max_new_tokens = self.config_obj.generation.max_new_tokens
+        # If an adapter config is provided, we want to wrap the model with a PEFT model
+        # for fine-tuning.
+        if self.config_obj.adapter:
+            from peft import get_peft_config, get_peft_model
+
+            # If the adapter config specifies a tokenizer name or path, it must match the model name
+            # Otherwise, we will set the tokenizer name or path to the model name
+            if (
```

Comment:
I would just take this out of the user's control. Don't let them specify it in the config, and always set it to model name. We can do this in a follow-up, though.

Reply:
I agree, Done!